File: SISTEMA DE TICKET/SISTEMA DE TICKET DIEGO/Tecnico.java
public class Tecnico extends Persona {
    private String strEspecialidad;
    private int intCargaActual;
    private int intCargaMaxima;

    public Tecnico(String strNombre, String strCorreo, String strTelefono, String strEspecialidad, int intCargaMaxima) {
        super(strNombre, strCorreo, strTelefono);
        this.strEspecialidad = strEspecialidad;
        this.intCargaMaxima = intCargaMaxima;
        this.intCargaActual = 0;
    }

    public boolean tieneCapacidad() { return intCargaActual < intCargaMaxima; }
    public void aumentarCarga() { if (tieneCapacidad()) intCargaActual++; }
    public void reducirCarga() { if (intCargaActual > 0) intCargaActual--; }
    public int getIntCargaActual() { return intCargaActual; }
}

File: SISTEMA DE TICKET/Proyectoticket/Tecnico.cs
using System;

namespace Proyectoticket.logica
{
    public class Tecnico : Persona
    {
        public string StrEspecialidad { get; set; }
        public int IntCargaActual { get; set; }
        public int IntCapacidadMaxima { get; set; }

        public Tecnico(string strCodigo, string strNombre, string strCorreo, string strEspecialidad, int intCapacidadMaxima)
            : base(strCodigo, strNombre, strCorreo)
        {
            this.StrEspecialidad = strEspecialidad;
            this.IntCargaActual = 0;
            this.IntCapacidadMaxima = intCapacidadMaxima;
        }

        public override string ObtenerRol()
        {
            return "Tecnico";
        }

        public bool EstaDisponible()
        {
            return BlnActivo && IntCargaActual < IntCapacidadMaxima;
        }

        public bool PuedeAtender(string strCategoria)
        {
            return EstaDisponible() &&
                   (StrEspecialidad.Equals(strCategoria, StringComparison.OrdinalIgnoreCase) ||
                    StrEspecialidad.Equals("General", StringComparison.OrdinalIgnoreCase));
        }

        public void AumentarCarga()
        {
            if (IntCargaActual < IntCapacidadMaxima)
            {
                IntCargaActual++;
            }
        }

        public void LiberarCarga()
        {
            if (IntCargaActual > 0)
            {
                IntCargaActual--;
            }
        }

        public override void MostrarInformacion()
        {
            base.MostrarInformacion();
            Console.WriteLine($" Especialidad: {StrEspecialidad} | Carga: {IntCargaActual}/{IntCapacidadMaxima}");
        }
    }
}